database/connection.go

```diff
@@ -17,3 +24,20 @@ func NewPool(url string) (*pgxpool.Pool, *sqlc.Queries) {
 
 	return pool, queries
 }
+
+func RunMigrations(databaseURL string) {
+	u, err := url.Parse(databaseURL)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Invalid database URL")
```

Comment:
Does `Err(err).Msg()` work? I think it might be preferable to just send the error itself, without the extra redundant message.

```suggestion
		log.Fatal().Err(err).Send()
```